### ISSUE TITLE
Icon: Use a smart stroke colour when none is defined

### ISSUE THREAD

**Issue by sarahmonster** [octopusthink/nautilus#92]
When we define a fillColor but no strokeColor, the stroke appears in the CSS currentColor (generally black):

<img width="185" alt="Screenshot 2019-06-26 at 02 20 11" src="https://user-images.githubusercontent.com/376315/60144094-121ac900-97b9-11e9-831e-0815a0b9ff3a.png">

I considered hiding the stroke entirely if no colour value had been passed, but a lot of these icons don't make sense without a stroke:

<img width="169" alt="Screenshot 2019-06-26 at 02 22 57" src="https://user-images.githubusercontent.com/376315/60144169-54440a80-97b9-11e9-9fb0-891503795a94.png">

A more elegant and visually pleasing solution here might be to use the provided fillColour as a base for the stroke colour, creating an effect like this:
<img width="177" alt="Screenshot 2019-06-26 at 02 20 16" src="https://user-images.githubusercontent.com/376315/60144095-121ac900-97b9-11e9-9041-164a7cda44e6.png">

Basically our options here are:
- fill render in fillColour, stroke render slightly darker
- stroke render in fillColour, fill render slightly lighter
- or some combination of the above, depending on the darkness of the provided fillColour value
